examples/docs_beta_snippets/docs_beta_snippets_tests/snippet_checks/guides/components/my-existing-project/README.md

```diff
@@ -0,0 +1,47 @@
+# my_existing_project
+
+This is a [Dagster](https://dagster.io/) project scaffolded with [`dagster project scaffold`](https://docs.dagster.io/guides/build/projects/creating-a-new-project).
```

Comment:
I feel like we don't want any mention of `dagster project scaffold`-- but does the content of this even show up anywhere?

Reply:
No, this is just a scaffolded location I added to git. Let me just drop the readme.